### PR TITLE
Add MediaConch to mcp-c osdeps

### PR DESCRIPTION
**Pull request by scollazo**

### CODE REVIEW

**Comment by scollazo** [comment]
A test run of the acceptance-tests in a centos system deployed using ansible, was failing because mediaconch wasn't installed.

File: src/MCPClient/osdeps/CentOS-7.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
   { "name": "libvpx", "state": "latest"},
   { "name": "md5deep", "state": "latest"},
   { "name": "mediainfo", "state": "latest"},
+  { "name": "mediconch", "state": "latest"},
```
Comment (sevein) [inline src/MCPClient/osdeps/CentOS-7.json]: typo?

**Review by sevein — CHANGES_REQUESTED** [review]

File: src/MCPClient/osdeps/CentOS-7.json (hunk 1 above)
Comment (scollazo) [reply src/MCPClient/osdeps/CentOS-7.json]: Indeed. It's fixed now.

**Review by jrwdunham — APPROVED** [review]
Once the typo is fixed this is good to go I think. (As scollazo knows, this also requires a PR re: https://github.com/artefactual-labs/ansible-archivematica-src/blob/qa/1.7.x/tasks/pipeline-osdeps-MCPClient.yml#L30)

**Comment by sevein** [comment]
I've filed an issue: https://github.com/artefactual/archivematica/issues/1032.

**Comment by sevein** [comment]
@scollazo can you include it in qa/1.x?

**Comment by scollazo** [comment]
cherrypicked as b11b138e94398a0979b730303c9b60aeb62901d4

**Pull request merged**